### PR TITLE
Adds basic version feature for traffic_layout info

### PR DESCRIPTION
**Pull request by zwoop**

### CODE REVIEW

**Comment by zwoop** [comment]
E.g.

```
munin (18:08) 453/0 $ ./src/traffic_layout/.libs/traffic_layout info --versions
traffic-server: 9.0.0
openssl: 0x1010102f
openssl_str: OpenSSL 1.1.1b  26 Feb 2019
```

File: src/traffic_layout/info.cc [hunk 1]
```diff
@@ -152,3 +155,19 @@ produce_layout(bool json)
     printf("}\n");
   }
 }
+
+void
+produce_versions(bool json)
+{
+  using LBW = ts::LocalBufferWriter<256>;
```
Comment (zwoop) [inline src/traffic_layout/info.cc]: @SolidWallOfCode Made me do this crazy, I'm sorry.

**Comment by zwoop** [comment]
I'll fix the missing symbols stuff, but give me a little bit of time, this is a good exercise to try to identify build issues and CI issues, where e.g. hwloc clearly ought to be available everywhere on our builds.

**Comment by ywkaras** [comment]
OK this will work too.

**Comment by SolidWallOfCode** [comment]
We will want to fix `print_var` at some point to be able to use BW formatting directly, rather than having to mess around with local variables and `.view()`.

**Pull request merged**